<ui version="4.0" >
 <class>IndirectDiffractionReduction</class>
 <widget class="QWidget" name="IndirectDiffractionReduction" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Indirect Diffraction</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QGroupBox" name="gbDemon" >
     <property name="title" >
      <string>DEMON</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QGridLayout" name="gridLayout_3" >
        <item row="0" column="0" >
         <widget class="QLabel" name="set_lbInst" >
          <property name="text" >
           <string>Instrument</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="MantidQt::MantidWidgets::InstrumentSelector" name="set_cbInst" >
          <property name="techniques" >
           <stringlist>
            <string>TOF Indirect Geometry Spectroscopy</string>
           </stringlist>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="set_lbReflection" >
          <property name="text" >
           <string>Reflection</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QComboBox" name="set_cbReflection" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <widget class="MantidQt::MantidWidgets::MWRunFiles" name="dem_rawFiles" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
            <horstretch>0</horstretch>
            <verstretch>41</verstretch>
           </sizepolicy>
          </property>
          <property name="label" stdset="0" >
           <string>Run Number</string>
          </property>
          <property name="multipleFiles" stdset="0" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="dem_ckSumFiles" >
          <property name="text" >
           <string>Sum Files</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11" >
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Spectra Min</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="set_leSpecMin" />
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Spectra Max</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="set_leSpecMax" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" >
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" >
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Correct By:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dem_cbCorrection" >
         <item>
          <property name="text" >
           <string>Monitor</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Vanadium Run</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="MantidQt::MantidWidgets::MWRunFiles" name="dem_vanadiumFile" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>41</verstretch>
        </sizepolicy>
       </property>
       <property name="label" stdset="0" >
        <string>Vanadium File</string>
       </property>
       <property name="multipleFiles" stdset="0" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="dem_swGrouping" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="page" >
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <widget class="MantidQt::MantidWidgets::MWRunFiles" name="dem_calFile" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Maximum" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>41</verstretch>
          </sizepolicy>
         </property>
         <property name="label" stdset="0" >
          <string>Cal File</string>
         </property>
         <property name="multipleFiles" stdset="0" >
          <bool>false</bool>
         </property>
         <property name="optional" stdset="0" >
          <bool>true</bool>
         </property>
         <property name="fileExtensions" >
          <stringlist>
           <string>.cal</string>
          </stringlist>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <layout class="QVBoxLayout" name="verticalLayout_5" >
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" >
         <item>
          <widget class="QLabel" name="label_3" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="text" >
            <string>Grouping Policy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="dem_cbGrouping" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <item>
            <property name="text" >
             <string>Individual</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>All</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>N Groups</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="dem_lbNumGroups" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="text" >
            <string>Number of Groups</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="dem_leNumGroups" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ckNormByCurrent" >
     <property name="text" >
      <string>Normalise by Current</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="lbPlotType" >
       <property name="text" >
        <string>Plot Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbPlotType" >
       <item>
        <property name="text" >
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Contour</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Spectra</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_22" >
     <item>
      <widget class="QCheckBox" name="dem_ckVerbose" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string>Verbose</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="dem_ckSave" >
       <property name="text" >
        <string>Save Result</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10" >
     <item>
      <widget class="QPushButton" name="pbHelp" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string>?</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_14" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbRun" >
       <property name="text" >
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_11" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbManageDirs" >
       <property name="text" >
        <string>Manage Directories</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::MWRunFiles</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/MWRunFiles.h</header>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::InstrumentSelector</class>
   <extends>QComboBox</extends>
   <header>MantidQtMantidWidgets/InstrumentSelector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
